packages/core/src/Popover/index.ts

```
@@ -22,6 +22,7 @@ export {
 export {
   injectPopoverRootContext,
   default as PopoverRoot,
+  type PopoverOpenChangeReason,
```

ESLint

[error] Expected "PopoverOpenChangeReason" to come before "PopoverRoot".

(perfectionist/sort-named-exports)